tests/stdlib/trandom.nims

```diff
@@ -0,0 +1,12 @@
+import std/[random, sets]
```

Comment:
optional:
instead how about
```
# in tests/test_nimscript.nims
import stdlib/trandom
```
and use conditional `when defined(nimscript)` as needed to make tests pass

which increases test coverage without duplication

Reply:
I have removed `tests/stdlib/trandom.nims` and added `import stdlib/trandom` to `tests/test_nimscript.nims`.